src/test/java/com/wikia/webdriver/testcases/mercurytests/curatedcontenttests/NavigationTests.java

```diff
@@ -76,4 +76,26 @@ public void MercuryCuratedNavigationTests_002_navigateThroughSection() {
 
     UrlChecker.isPathContainedInCurrentUrl(driver, ROOT_PATH_SECTION + section.getTitle());
   }
+
+  // CCT09
+  @Test(groups = {"MercuryCuratedNavigationTests_003", "MercuryCuratedNavigationTests",
```

Comment:
here https://github.com/Wikia/selenium-tests/pull/759/files#diff-a2b0fe6d72e025d30e23fd85f7cc21d7R84 is defined method with the same group. Could you resolve conflict with @justnpT